### DISCUSSION TITLE
program does not open

### DISCUSSION THREAD

**Discussion by mikoceta** [OpenTabletDriver/OpenTabletDriver#2027] [Q&A] [answered]
![gfgfffgf](https://user-images.githubusercontent.com/98658607/151670085-55029dea-179c-46f0-8241-d7de6384ce81.png)

how do i open it?

**Comment by jamesbt365** [comment, marked as answer]
That is part of the source code, download the actual windows release [here](https://github.com/OpenTabletDriver/OpenTabletDriver/releases/download/v0.6.0.2/OpenTabletDriver.win-x64.zip), extract that to a folder and then run it.
